src/noises.jl

```diff
@@ -205,7 +242,7 @@ Return:
 
 """
 function simulate_scenarios(laws, dims::Tuple)
-
+    warn("decrecated call to simulate_scenarios")
```

Comment:
you ask for a tuple of information where 2 out of three can be found in laws, 
leading to more complex calling, unless I'm mistaken ?